neutral_generation/smart_convert.py

```diff
@@ -91,16 +105,53 @@ def regex_token_replace(sentence: str, token: str, replacement: str) -> str:
     return sentence
 
 
-def score(sentence: str) -> float:
+def score(sentence: str, stride: int) -> float:
```

Comment:
Can we add some tests for these functions? Without tests this is error-prone.

Reply:
Yes, definitely. I will make this a priority.